tests/api_jobs/test_sync.py

```diff
@@ -37,6 +39,31 @@ def test_handles_missing_files_gracefully(self):
         assert self._state.file("9")
 
 
+def test_MetadataSyncJob_has_default_timeout(mocker, homedir, session, session_maker):
+    api_client = mocker.patch("securedrop_client.logic.sdclientapi.API")
+    remote_user = factory.RemoteUser()
+    api_client.get_users = mocker.MagicMock(return_value=[remote_user])
+
+    job = MetadataSyncJob(homedir)
+    job.call_api(api_client, session)
+    assert api_client.default_request_timeout == job.DEFAULT_REQUEST_TIMEOUT
+
+
+def test_MetadataSyncJob_takes_overriden_timeout(mocker, homedir, session, session_maker):
+    api_client = mocker.patch("securedrop_client.logic.sdclientapi.API")
+    remote_user = factory.RemoteUser()
+    api_client.get_users = mocker.MagicMock(return_value=[remote_user])
+
+    os.environ["SDEXTENDEDTIMEOUT"] = str(TIMEOUT_OVERRIDE)  # environment value must be string
```

Comment:
One alternative strategy to consider for tests that modify env vars might be to mock `os.environ`, see https://github.com/freedomofpress/securedrop-workstation/blob/b31d0acdf35cf5248acbe2dcd9372483dfde424f/launcher/tests/test_util.py#L259-L266 for an example (not a blocking comment, just an observation).